modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/resolver/ParameterGroupArgumentResolver.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.runtime.resolver;
+
+import static org.mule.config.i18n.MessageFactory.createStaticMessage;
+import static org.mule.module.extension.internal.util.IntrospectionUtils.getAlias;
+import static org.mule.module.extension.internal.util.IntrospectionUtils.checkInstantiable;
+import static org.mule.module.extension.internal.util.IntrospectionUtils.getParameterFields;
+import static org.mule.module.extension.internal.util.IntrospectionUtils.getParameterGroupFields;
+import org.mule.api.MuleRuntimeException;
+import org.mule.extension.annotations.Parameter;
+import org.mule.extension.annotations.ParameterGroup;
+import org.mule.extension.runtime.OperationContext;
+
+import java.lang.reflect.Field;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class is the one that makes it possible to resolve arguments annotated with
```

Comment:
This class is the one that makes it possible to resolve arguments annotated  > Resolves arguments annotated